src/file_pipe_log/format.rs

```diff
@@ -93,127 +93,76 @@ pub(super) fn lock_file_path<P: AsRef<Path>>(dir: P) -> PathBuf {
 }
 
 /// In-memory representation of `Format` in log files.
-#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Default)]
 pub struct LogFileFormat {
-    version: Version,
-    data_layout: DataLayout,
-}
-
-impl Default for LogFileFormat {
-    fn default() -> Self {
-        Self {
-            version: Version::default(),
-            data_layout: DataLayout::NoAlignment,
-        }
-    }
+    pub version: Version,
+    pub alignment: u64,
```

Comment:
Recommend to supply a necessary annotations to the meaning of different value on `alignment`. Such as:
``` Rust
...
/// `0` => no alignment.
/// `_` => aligned block size.
/// Default: `0`.
pub alignment: u64,
...
```